nexus/external-api/output/nexus_tags.txt

```diff
@@ -73,6 +73,8 @@ login_saml                               POST     /login/{silo_name}/saml/{provi
 API operations found with tag "metrics"
 OPERATION ID                             METHOD   URL PATH
 silo_metric                              GET      /v1/metrics/{metric_name}
+timeseries_query                         POST     /v1/timeseries/query
+timeseries_schema_list                   GET      /v1/timeseries/schema
```

Comment:
I think this should be `schemas` instead of `schema` since it's a list endpoint. Since we're breaking the API here anyway, it's as good a time there will ever be to fix it.